boards/xtensa/nxp_adsp_imx8m/nxp_adsp_imx8m_defconfig

```diff
@@ -1,27 +1,20 @@
 # SPDX-License-Identifier: Apache-2.0
 
-CONFIG_MAIN_STACK_SIZE=3072
-
 CONFIG_SOC_SERIES_NXP_IMX8M=y
 CONFIG_SOC_MIMX8M_ADSP=y
 CONFIG_BOARD_NXP_ADSP_IMX8M=y
 
-CONFIG_GEN_ISR_TABLES=y
-CONFIG_GEN_IRQ_VECTOR_TABLE=n
-
-CONFIG_XTENSA_RESET_VECTOR=y
-
-CONFIG_XTENSA_USE_CORE_CRT1=y
-
-CONFIG_XTENSA_SMALL_VECTOR_TABLE_ENTRY=y
+# size of stack for initialization and main thread
+CONFIG_MAIN_STACK_SIZE=3072
+# size in bytes of a CPU d-cache line
+CONFIG_DCACHE_LINE_SIZE=128
```

Comment:
Suggestion: if we know the data cache line size will stay the same for all boards using the XTENSA-based i.MX8M SoC we can treat this as a SoC quirk and move it to the SoC files.

Reply:
I've kept it here since.. it was already here :)
The stack size is also here.. and other xtensa boards define it here:
```
boards/xtensa/intel_adsp_ace20_lnl/intel_adsp_ace20_lnl_defconfig:16:CONFIG_DCACHE_LINE_SIZE=64
boards/xtensa/intel_adsp_ace15_mtpm/intel_adsp_ace15_mtpm_defconfig:16:CONFIG_DCACHE_LINE_SIZE=64
boards/xtensa/intel_adsp_cavs25/intel_adsp_cavs25_tgph_defconfig:24:CONFIG_DCACHE_LINE_SIZE=64
boards/xtensa/intel_adsp_cavs25/intel_adsp_cavs25_defconfig:23:CONFIG_DCACHE_LINE_SIZE=64
boards/xtensa/nxp_adsp_imx8m/nxp_adsp_imx8m_defconfig:10:CONFIG_DCACHE_LINE_SIZE=128
boards/arm64/mimx93_evk/mimx93_evk_a55_sof_defconfig:25:CONFIG_DCACHE_LINE_SIZE=64
```

I realize now that for `nxp_adsp_imx8` and `nxp_adsp_imx8x` is in soc files, so I can move it there for imx8m also, to be consisted on `nxp_adsp`.

Reply:
ACK, up to you.

Reply:
I've sent a v2 where I changed the DCACHE_LINE_SIZE to soc/.